Cargo.lock

```diff
@@ -41,6 +41,12 @@ version = "0.21.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "414dcefbc63d77c526a76b3afcf6fbb9b5e2791c19c3aa2297733208750c6e53"
 
+[[package]]
+name = "base64"
+version = "0.22.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "72b3254f16251a8381aa12e40e3c4d2f0199f8c6508fbecb9d91f575e0fbb8c6"
+
```

Comment:
Question: Why are we bumping base64 when it just duplicates it in-tree compared to the rest of the lower-level `rustls` crates?

Reply:
It has been updated in the upstream crates, but we haven't been merging `Cargo.lock` updates as proposed by Dependabot in this repo.

Reply:
> we haven't been merging Cargo.lock updates as proposed by Dependabot in this repo.

In retrospect I think turning that on for this repo was probably a mistake. I'd be OK with removing it again.

Reply:
Why do you think so?

Reply:
> Why do you think so?

The update PRs sit without reviews/merge and so the bot updates just become churn notifications.

From my perspective the norms for reviewing those PRs hasn't been set in this repo, or written down explicitly. I personally don't feel enough ownership here to blanket +1 updates that don't affect MSRV like I might in `rustls/rustls` or `rustls/webpki`.

I remember @complexspaces pushing back on landing the automation and in retrospect I think it would have been better to work through some kind of policy/expectation-setting before landing the automation. Lesson learned :-)

Reply:
> I'd be OK with removing it again

Here's a PR for that: https://github.com/rustls/rustls-platform-verifier/pull/120

If nothing else it gives us a place to continue this discussion separate from the `cargo deny` issue.

Reply:
I hope no one feels bad about the current state of the lockfile/dependency updates, that wasn't my intention at all. I tend to not look at the PRs because my OSS time is already fairly limited.

I did push back when first proposed because it seemed weird to me to update dependencies via lockfile just because they had updated, since this is generally not what I do at work or personal projects. I tend to only do dependency upgrades when there's a new direct dependency and then make sure the lockfile is clean as an end result. 

This is definitely something I'd be happy discussing more though, since I don't think there's one correct way to do this across all projects.

Reply:
> I hope no one feels bad about the current state of the lockfile/dependency updates

No hard feelings here!

Reply:
Nor here!

Reply:
@complexspaces I've tacked on a separate commit here which just did `cargo update`, which shows that this crate only depends on base64 0.22 after these changes.